src/api/history/history.controller.js

```diff
@@ -0,0 +1,211 @@
+const data = {
```

Comment:
Lets move this data into a seed file like FAQ.

Also create a DB model and retrieve the history day from the DB. (Similar to FAQ.)

Reply:
I was going to bring up this part. @AndCotOli Let me know if you want me to work with you on this.

Reply:
@spiray Of course, if you find more interesting data or you want to do the model and seed stuff you can contribute 🙃